lib/Yield/Assembly/PTML.cpp

```diff
@@ -327,3 +327,33 @@ std::string yield::ptml::controlFlowNode(const MetaAddress &Address,
 
   return Result;
 }
+
+template<bool IsFake>
+std::string yield::ptml::functionLink(const MetaAddress EntryAddress,
+                                      const model::Binary &Binary) {
+  if (EntryAddress.isInvalid())
+    return "";
+
+  auto Iterator = Binary.Functions.find(EntryAddress);
+  revng_assert(Iterator != Binary.Functions.end());
```

Comment:
Don't we have `.at`?

Reply:
Not sure, but I kind of prefer `find` to `at` because having an explicit assert leads to cleaner error messages (you see what's broken in the needed frame right away, not inside the container).

Reply:
Uhm, this is not a very strong argument. By this logic we should drop `.at` everywhere, while it's quite effective in hiding unnecessary iterators.

Reply:
Ah, wait, this case is somewhat special, there are two uses for the iterator, hence switching to `at` would possibly (if compiler can't catch and optimize it) cause the lookup to happen twice instead.